hedera-node/test-clients/src/main/resources/spec-default.properties

```diff
@@ -46,11 +46,11 @@ default.payer.pemKeyLoc=src/main/resources/genesis.pem
 #default.payer.pemKeyLoc=stabletestnet-account50.pem
 #default.payer.pemKeyLoc=mainnet-account950.pem
 default.payer.pemKeyPassphrase=swirlds
-#default.realm=12
-default.realm=0
+default.realm=12
+#default.realm=0
 default.receiverSigRequired=false
-#default.shard=11
-default.shard=0
+default.shard=11
+#default.shard=0
```

Comment:
Is this intentional? When we enabled nonzero shard/realm checks in CI, these defaults were left as is so that embedded tests would still run with `0.0`. That said, it might be better to explicitly set these values for `testEmbedded` in the gradle build file. 

In any event, it's not related to the contents of this PR, so no need to block on this

Reply:
Reverted, didn't mean to include.